src/Microsoft.ML.ImageAnalytics/VectorToImageTransform.cs

```diff
@@ -84,33 +99,37 @@ internal bool TryUnparse(StringBuilder sb)
             {
                 Contracts.AssertValue(sb);
                 if (ContainsAlpha != null || ContainsRed != null || ContainsGreen != null || ContainsBlue != null || ImageWidth != null ||
-                    ImageHeight != null || Offset != null || Scale != null || InterleaveArgb != null)
+                    ImageHeight != null || Offset != null || Scale != null || Interleave != null || Order != null || DefaultAlpha != null ||
+                    DefaultBlue != null || DefaultGreen != null || DefaultRed != null)
```

Comment:
Order?

Reply:
What Order?

---
In reply to: [259549474](https://github.com/dotnet/machinelearning/pull/2602#discussion_r259549474) [](ancestors = 259549474)

Reply:
Sorry I saw you added it :) 

---
In reply to: [259549716](https://github.com/dotnet/machinelearning/pull/2602#discussion_r259549716) [](ancestors = 259549716,259549474)